src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4410,7 +4410,7 @@ .ForwardHelpCategory Cmdlet
             // Variable which controls the encoding for piping data to a NativeCommand
```

Comment:
It would be great if you can add more comments about the characteristics of this encoding and mention that's why we use it as the default output encoding.